src/components/download/LegendPosition.js

```diff
@@ -10,64 +11,18 @@ export const legendPositions = [
     'bottomright',
 ];
 
-const styles = theme => ({
-    root: {
-        width: 150,
-        paddingLeft: 30,
-    },
-    label: {
-        fontSize: 14,
-        padding: 5,
-    },
-    position: {
-        position: 'relative',
-        display: 'inline-block',
-        backgroundColor: theme.palette.background.default,
-        width: 50,
-        height: 50,
-        margin: '1px 5px',
-        outline: `1px solid ${theme.palette.divider}`,
-        cursor: 'pointer',
-    },
-    legend: {
-        position: 'absolute',
-        width: 12,
-        height: 12,
-        backgroundColor: theme.palette.action.active,
-    },
-    selected: {
-        outline: `3px solid ${theme.palette.primary.main}`,
-    },
-    topleft: {
-        top: 8,
-        left: 8,
-    },
-    topright: {
-        top: 8,
-        right: 8,
-    },
-    bottomleft: {
-        bottom: 8,
-        left: 8,
-    },
-    bottomright: {
-        bottom: 8,
-        right: 8,
-    },
-});
-
-export const LegendPosition = ({ position, onChange, classes }) => (
-    <div className={classes.root}>
-        <div className={classes.label}>{i18n.t('Legend position')}</div>
+export const LegendPosition = ({ position, onChange }) => (
+    <div className={styles.root}>
+        <div className={styles.label}>{i18n.t('Legend position')}</div>
```

Comment:
From a HTML semantics perspective I'd consider using something else than a `div` to wrap the label

Reply:
Switched to <label> element https://html.spec.whatwg.org/multipage/forms.html#the-label-element